textproc/par/Portfile

```diff
@@ -1,12 +1,10 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-# Based on: http://www.openbsd.org/cgi-bin/cvsweb/ports/textproc/par/Makefile
-
 PortSystem          1.0
+PortGroup           makefile 1.0
 
 name                par
-version             1.52
-revision            2
+version             1.53.0
```

Comment:
Revision should be set to 0 rather than removed...

Reply:
Added back.